tests/inference/qnn_test.py

```diff
@@ -441,22 +451,33 @@ def test_expectation_bitstring_energy(self):
         n_moments=n_moments,
         p=act_fraction)
     hamiltonian_raw_circuit = hamiltonian_raw_circuits[0]
-    hamiltonian_circuit = models.DirectQuantumCircuit(hamiltonian_raw_circuit)
+    hamiltonian_circuit_initializer = tf.keras.initializers.RandomUniform(
+        minval=qnn_minval, maxval=qnn_maxval, seed=self.tf_random_seed_alt)
+    hamiltonian_circuit = models.DirectQuantumCircuit(
+        hamiltonian_raw_circuit, hamiltonian_circuit_initializer)
+    hamiltonian_circuit.build([])
```

Comment:
is this build needed? is the reason we need it now so that the initializer is constructed? is this something we can eventually automate, like before?

Reply:
definitely fair to build and keep as-is - i recall you had a PR which automated building of member classes. that was different because you built members rather than the class itself which makes for a reasonable distinction